als-suggestions/shared/src/test/resources/test/oas30/by-directory/yaml/open-api-document/paths/path-object/operation/responses/links/parameters/expected/parameters-request.yaml.json

```diff
@@ -15,11 +15,11 @@
     "textEdit": {
       "range": {
         "start": {
-          "line": 14,
+          "line": 15,
```

Comment:
Are you sure this is expected?
LSP position lines are 0 based, so the text edit will be applied:
`      parameters:
          - 
            *`
Instead of at the sequence line, or I missed something?

Reply:
Good sight! You were right, this test was failing. It's was my mistake when I was trying to prove something else and forgot to change it back.
Thanks @pope1838